inspec.yml

```diff
@@ -30,13 +30,10 @@ inputs:
 
   - name: temporary_account_period # V-92975
     desc: "List the number of days that temporary accounts remain active for"
-    type: Numeric
-    value: 3
-
-  - name: temporary_account_period_phrase # V-92975
-    desc: "List the number of hours/days that temporary accounts remain active for"
-    type: String
-    value: "72 hours"
+    type: Hash
+    value: 
+      hours: 72
+      days: 3
```

Comment:
I personally love using a `Hash` to use a map when needed - and this would work correctly for the baseline profile. However, it would clash with the input data type in our CMS overlays that pull from this baseline. The simplest update would be to use `Numeric` and perform arithmetic calculations during input evaluation (like you did in V-93509). A less simple update would be to update the overlay inputs as well to be consistent with the baseline. @ejaronne thoughts?

Reply:
Discussed this with Eugene and we have decided to continue using `Numeric` for now. We may also need to update the ARS 5.0 overlay but I'll create another issue for that.

Reply:
Issue created https://github.com/CMSgov/cms-ars-5.0-microsoft-windows-server-2019-stig-overlay/issues/7
The value in the describe block for `V-92975` seems to have been changed accidentally.